docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
+
+```js
+var I13nUtils = require('react-i13n').I13nUtils;
+var DemoComponent = React.createClass({
+    mixins: [I13nUtils],
+    displayName: 'DemoComponent',
+    render: function() {
+        // this.getI13nNode() to access the i13nNode created by createI13nNode
```

Comment:
it's made for mail team, but actually some users asked me about some case that for example slideshow, we can do something like


```
var I13nUtils = require('react-i13n').I13nUtils;
var SlideShow = React.createClass({
    mixins: [I13nUtils],
    displayName: 'DemoComponent',
    handleSwipe: function() {
        // fire beacon, here using this.getI13nNode(), and it can still inherit the data of its parent
    }
});

var I13nSlideShow = createI13nNode(SlideShow);


// on some other component
<SlideShow i13nModel={i13nModel}/>
```